gdx/src/com/badlogic/gdx/math/Interpolation.java

```diff
@@ -27,24 +27,91 @@ public float apply (float start, float end, float a) {
 		return start + (end - start) * apply(a);
 	}
 
-	//
+	/** Speeds are expressed in terms of units of "total change" over "duration". To prepare inputs for this method, you may use:
+	 * <p>
+	 * {@code speed = worldSpeed * duration / (end - start)}
+	 * @param startSpeed Beginning rate of change. Used only by {@linkplain SplineInterpolation SplineInterpolations}.
+	 * @param endSpeed Ending rate of change. Used only by {@linkplain SplineInterpolation SplineInterpolations}.
+	 * @param a Alpha value between 0 and 1, where 1 maps to the total duration.
+	 * @return The current rate of change. To convert this to world speed, you may use:
+	 *         <p>
+	 *         {@code worldSpeed = speed * (end - start) / duration} */
+	public abstract float speed (float startSpeed, float endSpeed, float a);

```

Comment:
Not super excited about every Interpolation needing to implement a second method.